py-polars/tests/unit/operations/test_aggregations.py

```diff
@@ -282,12 +282,14 @@ def test_sum_empty_and_null_set() -> None:
     assert df.groupby("b").agg(pl.sum("a"))["a"].item() == 0.0
 
 
-def test_10455(monkeypatch: Any) -> None:
-    monkeypatch.setenv("POLARS_FORCE_PARTITION", "1")
+def test_null_sum_streaming_10455() -> None:
     df = pl.DataFrame(
         {
-            "x": [1] * 100,
-            "y": [None] * 100,
+            "x": [1] * 10,
```

Comment:
A small number of rows, splits the chunks of length 1. Length 1 is sorted, so this triggers a sorted fast path for some of the chunks.